src/hidimstat/noise_std.py

```diff
@@ -104,6 +104,7 @@ def reid(
         cv=cv,
         tol=tolerance,
         max_iter=max_iterance,
+        random_state=seed + 1,
```

Comment:
```suggestion
        random_state=seed,
```
What if the seed is a generator? Should we rename the argument of `reid` seed --> random_state for consistency?

Reply:
For renaming, it will be done in PR #368 or #389.